client_test.go

```diff
@@ -0,0 +1,26 @@
+package main
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/go-redis/redis"
```

Comment:
Fixed.

`dep` knows can recognize test dependencies and treats them accordingly.